src/main/scala/controllers/OpportunityController.scala

```diff
@@ -30,10 +30,9 @@ class OpportunityController @Inject()(opportunities: OpportunityOps, application
 
   def showOpportunityPreview(id: OpportunityId, sectionNumber: Option[Int]) = Action.async {
     val f1 = OptionT(opportunities.byId(id))
-    val f2 = OptionT(applications.byOpportunityId(id))
 
-    (for (o <- f1; a <- f2) yield (o, a)).value.map {
-      case Some((o, a)) => Ok(views.html.opportunityPreview(a.id, o, sectionNumber.getOrElse(1)))
+    (for (o <- f1) yield o).value.map {
```

Comment:
In fact you no longer need the OptionT here. They're only used to deal with flatmapping two `Future[Option[_]]`s. Just make the call to `opportunities.byId(id)` and map it.